src/pybind/mgr/dashboard/module.py

```diff
@@ -417,6 +419,22 @@ def set_rgw_credentials(self):
 
         return 0, 'RGW credentials configured', ''
 
+    @CLIWriteCommand("dashboard set-login-banner")
```

Comment:
Shouldn't we have an `unset-login-banner` or a `reset-login-banner` command too inorder to reset the login-banner to an empty text if that's what user wants. I tried to pass an empty file like
`ceph dashboard set-login-banner -i login.txt` and it basically gave me this error. 
`Error EINVAL: Please specify the login banner file with "-i" option`. 

I can pass a file with just a space but it creates an unnecessary <p> tag in the login page to show just a space character.

Reply:
added `unset-login-banner` command